components/formats-gpl/src/loci/formats/in/NativeND2Reader.java

```diff
@@ -1983,6 +1992,10 @@ else if (name.equals("dPosX")) {
     catch (Exception e) {
       LOGGER.debug("", e);
     }
+
+    if (zHigh != null && zLow != null && trueSizeZ != null) {
```

Comment:
For safety I would check that trueSizeZ > 0 so as to avoid a divide by 0 error

Reply:
Fixed in 7d8d556